Deploy-CAM.ps1

```diff
@@ -331,7 +331,8 @@ function New-UserStorageAccount {
         $location
     )
 
-    $saName = -join ((97..122) | Get-Random -Count 18 | % {[char]$_})
+    $saName = -join ((97..122) | Get-Random -Count 16 | % {[char]$_})
+    $saName = 'cam0' + $saName
```

Comment:
why cam0?

Reply:
Needed some prefix that had only lower case letters and numbers... Nice restrictive naming!